### ISSUE TITLE
Flatpak-VLC: open a video, close it immediately and open another video broke VLC.

### ISSUE THREAD

**Issue by valeriodean** [flathub/org.videolan.VLC#47]
**Note:**
This issue was originally reported to flatpak, [https://github.com/flatpak/flatpak/issues/2208#issue-367577331](url)

**Linux distribution and version**
Fedora 28

**Flatpak version**
1.0.3

**Description of the problem**
I use VLC through flatpak.
The problem I noticed appear when I open a video, recognise it is the wrong one and close it immediately, then click on another video and... nothing happen.
VLC seems dead.
When I check the system activity there are vlc related program in execution, but clicking on a video file doesn't trigger the player (see attached screenshot) or the player doesn't work any more.
To solve the problem I need to kill those process.
![vlc_flatpack_problem](https://user-images.githubusercontent.com/9042830/46622737-dc9b5600-cb2b-11e8-95a2-428e71c114c8.png)

Note: if you do not change the video file and try to open again the same file you closed one second ago, it works. If you pick another video, it doesn't work.

**Steps to reproduce**
open video A and close VLC after some seconds
open video B... nothing happens
kill the vlc related process
open video B and vlc starts without problem.

**Request**
VLC in flatpak format should manage the workflow above without problems as well as the "legacy" installation.

**Comment by MatMaul** [comment]
Please try `flatpak run org.videolan.VLC -vvv your_video.mp4` and quit the video. According to what you say it should not quit. Please paste the log to see if we can see something there.

It is happening with all videos ? it's weird, I was on Fedora 28 a week ago (29 now) and I didn't have this kind of problem.

**Comment by MatMaul** [comment]
Please give more infos, I can't reproduce. Full logs please, and software and hardware specs.
Don't bother testing the launch of a second VLC, the problem is that the first one doesn't quit.

Also please test with the packaged VLC version of your distribution and post the logs (with -vvv to have full logs).

**Comment by Unrud** [comment]
> Please give more infos

The problem occurs with all videos with 60 fps that I tested. Videos with 50 fps work fine. It's always reproducible. The codec doesn't matter.

> Full logs please

There is nothing useful in there. I compared the log output of two videos that are identical besides the framerate and the log output is basically the same until the line ``main playlist debug: incoming request - stopping current input``.

> software

Fedora Silverblue 29

> hardware specs

Intel Haswell with HD 4600

>  test with the packaged VLC version of your distribution

Same problem. I guess it's not related to Flatpak and the issue can be closed.
